lib/matplotlib/pyplot.py

```diff
@@ -2510,203 +2428,83 @@ def spy(Z, precision=0, marker=None, markersize=None, aspect='equal', **kwargs):
 
 # Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
 @_autogen_docstring(Axes.acorr)
-def acorr(x, hold=None, data=None, **kwargs):
+def acorr(x, **kwargs):
     ax = gca()
-    # Deprecated: allow callers to override the hold state
-    # by passing hold=True|False
-    washold = ax._hold
-
-    if hold is not None:
-        ax._hold = hold
-        from matplotlib.cbook import mplDeprecation
-        warnings.warn("The 'hold' keyword argument is deprecated since 2.0.",
-                      mplDeprecation)
-    try:
-        ret = ax.acorr(x, data=data, **kwargs)
-    finally:
-        ax._hold = washold
+    ret = ax.acorr(x, data=data, **kwargs)
 
     return ret
 
 # Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
 @_autogen_docstring(Axes.angle_spectrum)
 def angle_spectrum(x, Fs=None, Fc=None, window=None, pad_to=None, sides=None,
-                   hold=None, data=None, **kwargs):
+                   **kwargs):
```

Comment:
Note to future me: the data kwarg shouldn't have gone here, this needs fixing. (or if anyone fancies force pushing a fix to my branch)